com/win32com/src/oleargs.cpp

```diff
@@ -138,7 +140,9 @@ BOOL PyCom_VariantFromPyObject(PyObject *obj, VARIANT *var)
     }
     else if (PyLong_Check(obj)) {
         int sign = _PyLong_Sign(obj);
-        size_t nbits = _PyLong_NumBits(obj);
+        std::bitset<64> bs(PyLong_AsUnsignedLongLong(obj));
+        size_t nbits = bs.count();
+        // size_t nbits = _PyLong_NumBits(obj);
```

Comment:
`_PyLong_NumBits` has been restored and a public method replacement is planned. Should this be kept as it was? Replacing with the public version once available? (Probably behind a python version check)

Reply:
I think this sounds fine, but it should be the other way around - ie, let's revert this change for this PR, and experimentation can happen in future PRs once this lands?

Reply:
Sure

Reply:
Of course, we'll have to wait until the next 3.13 release for the CI builds to not spew errors

Reply:
damn, I missed that subtlety - for some reason I assumed there *was* a new version with this. Do you have any idea of when that's likely to be.

Reply:
Beta 2 should be today: https://peps.python.org/pep-0719/#schedule
Assuming https://github.com/python/cpython/pull/119970 is included (since it was yesterday)